lib/actions/google/drive/sheets/google_sheets.js

```diff
@@ -20,7 +20,9 @@ const MAX_REQUEST_BATCH = process.env.GOOGLE_SHEETS_WRITE_BATCH ? Number(process
 const MAX_ROW_BUFFER_INCREASE = 6000;
 const SHEETS_MAX_CELL_LIMIT = 5000000;
 const MAX_RETRY_COUNT = 5;
+const RETRY_BASE_DELAY = process.env.GOOGLE_SHEETS_BASE_DELAY ? Number(process.env.GOOGLE_SHEETS_WRITE_BATCH) : 3;
```

Comment:
it could be helpful to include the time unit in the constant name

Reply:
+1 but can be done as a fast followup